packages/manager/modules/iam/src/resolves/misc.resolve.js

```diff
@@ -38,13 +38,35 @@ alertResolve.key = 'alert';
  *   type: string
  * }|null}
  */
-const entityResolve = /* @ngInject */ (policy) => {
-  if (policy) return { data: policy, type: ENTITY.POLICY };
-  return null;
+const entityResolve = /* @ngInject */ (policy, identity) => {
+  let entity = null;
+  if (identity && policy) {
+    const [, userName] = identity.components;
+    entity = {
+      data: { policy, identity, name: userName },
+      type: ENTITY.IDENTITY,
+    };
+  } else if (policy) {
+    entity = { data: policy, type: ENTITY.POLICY };
+  }
+  return entity;
 };
 
 entityResolve.key = 'entity';
-entityResolve.resolves = [policyParamResolve];
+entityResolve.resolves = [policyParamResolve, identityParamResolve];
+
+// ---------------------------------------------------------------------------------------------------- //
+
+/**
+ * if statement form is needed for delete entity
```

Comment:
Resolves do not know about which component use them
```suggestion
 * Whether the entity requires a statement 
```

Reply:
It's applied